mainnet/draft_genesis.json

```diff
@@ -0,0 +1,682 @@
+        "reward_portion_for_stakers": "0.250000000000000000",
+        "max_eden_reward_apr_lps": "0.500000000000000000",
+        "supported_reward_denoms": [],
+        "protocol_revenue_address": "NEED THIS ADRESS"
+      },
+      "external_incentives": [],
+      "external_incentive_index": "0",
+      "pool_infos": [],
+      "pool_reward_infos": [],
+      "user_reward_infos": [],
+      "pool_rewards_accum": []
+    },
+    "oracle": {
+      "params": {
+        "band_channel_source": "channel-1",
+        "oracle_script_id": "37",
+        "multiplier": "18",
+        "ask_count": "4",
+        "min_count": "3",
+        "fee_limit": [
+          {
+            "denom": "uband",
+            "amount": "30"
+          }
+        ],
+        "prepare_gas": "600000",
+        "execute_gas": "600000",
+        "client_id": "band_price_id",
+        "band_epoch": "band_epoch",
+        "price_expiry_time": "86400",
+        "life_time_in_blocks": "1"
+      },
+      "port_id": "oracle",
+      "asset_infos": [
+        {
+          "denom": "uelys",
+          "display": "ELYS",
+          "band_ticker": "ELYS",
+          "elys_ticker": "ELYS",
+          "decimal": "6"
+        },
+        {
+          "denom": "ueden",
+          "display": "EDEN",
+          "band_ticker": "EDEN",
+          "elys_ticker": "EDEN",
+          "decimal": "6"
+        },
+        {
+          "denom": "uedenb",
+          "display": "EDENB",
+          "band_ticker": "EDENB",
+          "elys_ticker": "EDENB",
+          "decimal": "6"
+        }
+      ],
+      "prices": [],
+      "price_feeders": [
+        {
+          "feeder": "elys10d07y265gmmuvt4z0w9aw880jnsr700j6z2zm3",
+          "is_active": true
+        },
+        {
+          "feeder": "elys16qgewtplahkqwqa0aqv4pxnxa58ulu48k6crhj",
+          "is_active": true
+        },
+        {
+          "feeder": "elys1zwexzk6ns5ermvag5fc0gtyvrnxyaz9kzaflqf",
+          "is_active": true
+        }
+      ]
+    },
+    "parameter": {
+      "params": {
+        "min_commission_rate": "0.050000000000000000",
+        "max_voting_power": "0.66",
+        "min_self_delegation": "1",
+        "total_blocks_per_year": "6307200",
+        "rewards_data_lifetime": "86400"
+      }
+    },
+    "params": null,
+    "perpetual": {
+      "params": {
+        "leverage_max": "25.000000000000000000",
+        "borrow_interest_rate_max": "0.300000000000000000",
+        "borrow_interest_rate_min": "0.100000000000000000",
+        "borrow_interest_rate_increase": "0.000300000000000000",
+        "borrow_interest_rate_decrease": "0.000300000000000000",
+        "health_gain_factor": "0.000000220000000000",
+        "max_open_positions": "100000",
+        "pool_open_threshold": "0.650000000000000000",
+        "borrow_interest_payment_fund_percentage": "0.100000000000000000",
+        "borrow_interest_payment_fund_address": "elys1xd986tqfwwcxeywcmsvq62mwk4z8cvzsfnplt3",
```

Comment:
@avkr003 that address seems to be the same as in testnet

Reply:
its module address